app/views/public_pages/home.html.erb

```diff
@@ -25,6 +25,7 @@
 <% content_for :body_class, "body--home" %>
 
 <%= render 'shared/facebook_pixel' %>
+<%= render 'shared/adsrver_pixel' %>
```

Comment:
trade_desk_pixel might be easier to spell

Reply:
renamed! also actually added the credentials haha (though only to demo and prod)